externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkGrpcBackendService.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.engine.spark
+
+import org.apache.spark.sql.SparkSession
+
+import org.apache.kyuubi.engine.spark.session.SparkSQLSessionManager
+import org.apache.kyuubi.service.AbstractBackendService
+import org.apache.kyuubi.session.SessionManager
+
+/**
+ * A [[org.apache.kyuubi.service.BackendService]] constructed
+ * with [[SparkSession]] which give it the ability to talk with
+ * Spark and let Spark do all the rest heavy work :)
+ *
+ *  @param name  Service Name
+ *  @param spark A [[SparkSession]] instance
+ *               that this backend service holds to run [[org.apache.kyuubi.operation.Operation]]s.
+ */
+class SparkGrpcBackendService(name: String, spark: SparkSession)
```

Comment:
Do I understand correctly, that this class is not used at all and we just start SparkConnect gRPC server in the `SparkGrpcFrontendService`?